invenio_administration/assets/semantic-ui/js/invenio_administration/src/formik/GenerateForm.js

```diff
@@ -34,11 +41,14 @@ const generateFieldProps = (
   }
 
   const htmlDescription = (
-    <div
-      dangerouslySetInnerHTML={{
-        __html: formFieldConfig?.description || fieldSchema?.metadata?.description,
-      }}
-    />
+    <>
+      <p />
+      <div
+        dangerouslySetInnerHTML={{
+          __html: formFieldConfig?.description || fieldSchema?.metadata?.description,
```

Comment:
```suggestion
          __html: formFieldConfig?.description || fieldSchema.metadata?.description,
```

Even if `metadata` is not defined in `fieldSchema`, it will short-circuit and return `undefined` if the chain operator (`?`) is used only in `metadata?.description`